4.4.2/style.css

```diff
@@ -0,0 +1,77 @@
+
+.facebook-text {
+  width: 200px;
+  padding: 20px;
+  padding-top: 0;
+  font-family: Fira Code;
+  font-size: 1em;
+  color: white;
+}
+
+.instagram-text {
+  width: 200px;
+  padding: 20px;
+  padding-top: 0;
+  font-family: sans-serif;
+  font-size: 1em;
+  color: white;
+}
+
+.twitter-text {
+  width: 200px;
+  padding: 20px;
+  padding-top: 0;
+  font-family: monospace;
+  font-size: 1em;
+  color: white;
+}
```

Comment:
You're repeating a lot of styles, is better if you have a social-network class (could put it in the container instead of putting facebook-container, instagram-container ...), that have all those attributes, except font family, and then in the instagram-text, facebook-text ... you put the font family